### DISCUSSION TITLE
Avatar image broken

### DISCUSSION THREAD

**Discussion by hdin0923** [academicpages/academicpages.github.io#2165] [Q&A]
For all people that are struggling with uploading avatar images, it turns out that (I may be wrong but I tested thoroughly) the avatar image only accepts the exact image name "profile.png". It cannot be .jpg file, it cannot be something like "profileee.png". So in order to upload your avatar image correctly, **you need to upload an image and rename it to "profile.png" (replace the existing one)**. Also, **ensure that the image size is less than 1MB**.

**Comment by AliAkber123** [comment]
still not working

**Reply by sklasek** [reply]
I'm having the same issue as @AliAkber123. Even with an avatar pic named "profile.png" under 1 Mb, no image is displaying [here](https://sklasek.github.io/). Any suggestions?

**Reply by rjzupkoii** [reply]
@sklasek Looks like the `url` setting in your `_config.yml` is not set correctly. You also need to make sure the profile image in `author.avatar` is the `images` directory.

**Reply by sklasek** [reply]
@rjzupkoii thanks, I've resolved it!

**Comment by dongjh20** [comment]
thanks! perfectly solve my issue. wired design :)
